editor/editor_help.cpp

```diff
@@ -2650,7 +2811,69 @@ EditorHelpBit::EditorHelpBit() {
 	set_custom_minimum_size(Size2(0, 50 * EDSCALE));
 }
 
-//// FindBar ///
+/// EditorHelpTooltip ///
+
+void EditorHelpTooltip::_notification(int p_what) {
+	if (!tooltip_text.is_empty() && p_what == NOTIFICATION_POSTINITIALIZE) {
+		parse_tooltip(tooltip_text);
+	}
```

Comment:
Would be preferable to follow our standard switch style even if it's one case.

Reply:
The class above has this boilerplate for one single case as well, and this consistency helps when we modify existing logic as we don't need to touch the already existing case keeping the history and git blame clean.